pom.xml

```diff
@@ -39,181 +39,145 @@
-                <module>nucleus</module>
-                <module>appserver</module>
-                <module>api</module>
-            </modules>
-            <properties>
-                <maven.test.skip>true</maven.test.skip>
-                <project>payara</project> 
-            </properties>
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-source-plugin</artifactId>
-                        <version>${maven.source.plugin.version}</version>
                         <configuration>
                             <excludeResources>true</excludeResources>
                             <excludes>**/*.properties</excludes>
                             <excludes>**/*.jnlp</excludes>
                             <excludes>**/*.class</excludes>
                             <excludes>**/*.html</excludes>
                             <includePom>false</includePom>
+                            <finalName>payara-${project.version}</finalName>
+                            <includePom>false</includePom>
                         </configuration>
                         <executions>
                             <execution>
+                                <id>attach-sources-aggregate</id>
                                 <phase>pre-site</phase>
-                                <id>attach-sources</id>
+                                <inherited>false</inherited>
                                 <goals>
-                                    <goal>jar-no-fork</goal>
                                     <goal>aggregate</goal>
                                 </goals>
-                                <configuration>
-                                    <finalName>payara-5.193-SNAPSHOT</finalName>    
-                                    <includePom>false</includePom>                            
-                                </configuration>
                             </execution>
                         </executions>
                     </plugin>
                 </plugins>
             </build>
         </profile>
-             
+
         <profile>
             <id>javadoc</id>
             <activation>
-                <activeByDefault>false</activeByDefault>
                 <property>
                     <name>javadoc</name>
                     <value>true</value>
                 </property>
             </activation>
-            <modules>
-                <module>nucleus</module>
-                <module>appserver</module>
-                <module>api</module>
-            </modules>
-            <properties>
-                <maven.test.skip>true</maven.test.skip>
-            </properties>
             <build>
+                <finalName>payara-${project.version}</finalName>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I'm personally not fan of dropping groupId, there are enough rules to remember about maven already :)

Reply:
It's default, I think there is no need to remember any "new rule". You don't even use the groupId when you are looking for a documentation and you can even run build without that.
https://maven.apache.org/guides/introduction/introduction-to-plugin-prefix-mapping.html

On the command line the maven- and -plugins are defaults too, so usually you run commands simply like this:
`mvn dependency:build-classpath -pl :payara`